gcp_variant_transforms/libs/bigquery_util_test.py

```diff
@@ -460,6 +460,28 @@ def test_raise_error_if_dataset_not_exists(self):
                       bigquery_util.raise_error_if_dataset_not_exists,
                       client, 'project', 'dataset')
 
+  def test_get_table_base_name(self):
+    without_suffix1 = 'project_id.dataset_id.table_id'
+    without_suffix2 = 'project_id:dataset_id.table_id'
+    self.assertEqual(without_suffix1,
+                     bigquery_util.get_table_base_name(without_suffix1))
+    self.assertEqual(without_suffix2,
+                     bigquery_util.get_table_base_name(without_suffix2))
+
+    with_suffix1 = without_suffix1 + '___chr1'
```

Comment:
Please modify to `__chr1`, here and everywhere else below (ie. 2 underscores, if you are using SAMPLE_TABLE_SUFFIX_SEPARATOR).